engine/runtime/src/boxed.rs

```diff
@@ -205,6 +236,13 @@ impl<T: Bytes> Boxed<T> {
     fn is_locked(&self) -> bool {
         self.prot.get() == Prot::NoAccess
     }
+
+    #[cfg(test)]
+    #[allow(dead_code)]
+    /// Returns the address of the pointer to the data
+    pub fn get_ptr_address(&self) -> usize {
```

Comment:
Is this still being used? Also may be better to use `std::ptr::addr_of!` macro.

Reply:
This is used for testing. also `addr_of` won't give the exact address of where the data is stored since this is encapsulated into multiple structure with private fileds